parlai/agents/image_seq2seq/modules.py

```diff
@@ -228,14 +265,80 @@ def forward(
             A (full_enc, full_mask) tuple, which represents the encoded context
             and the mask
         """
+        if self.fusion is FusionType.LATE:
+            return self._forward_late_fusion(src_tokens, image_features)
+        elif self.fusion is FusionType.EARLY:
+            return self._forward_early_fusion(src_tokens, image_features)
+
+        raise RuntimeError(f'Unsupported fusion type: {self.fusion}')
```

Comment:
nit: I always think `else: raise RuntimeError` is cleaner, but this really doesn't matter :P

Reply:
happy to change